pkg/terminal/operator.go

```diff
@@ -42,3 +59,34 @@ func workspaceOperatorIsRunning() (bool, error) {
 	}
 	return true, nil
 }
+
+// operatorIsInstalled checks to see that a web-terminal-operator is installed on the cluster
+func operatorIsInstalled() (bool, error) {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return false, err
+	}
+
+	config.GroupVersion = OperatorGroupVersion
+	config.APIPath = "apis"
+
+	client, err := dynamic.NewForConfig(config)
+	if err != nil {
+		return false, err
+	}
+
+	_, err = client.Resource(*OperatorAPIResource).Namespace(operatorsNamespace).Get(context.TODO(), webTerminalOperatorName, metav1.GetOptions{})
+
+	// Web Terminal subscription is found
+	if err == nil {
```

Comment:
We should be checking for the error right after the call.
```suggestion
	if err != nil ||  errors.IsNotFound(err) {
		return false, err
	}

	return true, nil
```

Reply:
Sorry for getting back my :+1: 
I missed that the proposed way won't work as original. It will log error when subscription is not found.

it may be 
```go
	if err != nil {
      if errors.IsNotFound(err) {
        return false, nil
      }
	  return false, err
    }
```